### PR TITLE
Add countryCode setting for digital wallets

### PR DESCRIPTION
**Pull request by colinbird**
Added the `countryCode` for both Apple Pay and Google Pay

### CODE REVIEW

**Review by ashkarpetin — DISMISSED** [review]

**Review by dara123 — DISMISSED** [review]

File: openapi/components/schemas/PaymentCards/DigitalWallets.yaml [hunk 1]
```diff
@@ -16,6 +16,12 @@ properties:
         description: A string of 64 or fewer UTF-8 characters containing the canonical name for your store, suitable for display. Don’t localize the name.
         type: string
         example: "Test Merchant"
+      countryCode:
+        description: The country where the payment is processed.  This is often the location for settling the payment. Consult with your payment service provider (PSP) to determine the appropriate country code.
```
Comment (vykhrystiuk) [inline openapi/components/schemas/PaymentCards/DigitalWallets.yaml:20]:
```suggestion
        description: The country where the payment is processed. This is often the location for settling the payment. Consult with your payment service provider (PSP) to determine the appropriate country code.
```

File: openapi/components/schemas/PaymentCards/DigitalWallets.yaml [hunk 2]
```diff
@@ -34,3 +40,9 @@ properties:
         description: The merchant origin in Google Pay. The fully qualified domain name.
         type: string
         example: "www.example.com"
+      countryCode:
+        description: The country where the payment is processed.  This is often the location for settling the payment. Consult with your payment service provider (PSP) to determine the appropriate country code.
```
Comment (vykhrystiuk) [inline openapi/components/schemas/PaymentCards/DigitalWallets.yaml:44]:
```suggestion
        description: The country where the payment is processed. This is often the location for settling the payment. Consult with your payment service provider (PSP) to determine the appropriate country code.
```

File: openapi/components/schemas/ReadyToPay/Features/PaymentCardDigitalWalletFeatures/ApplePayFeature.yaml [hunk 3]
```diff
@@ -7,4 +7,10 @@ properties:
     description: A string of 64 or fewer UTF-8 characters containing the canonical name for your store, suitable for display. Don’t localize the name.
     type: string
     example: "Test Merchant"
+  countryCode:
+    description: The country where the payment is processed.  This is often the location for settling the payment. Consult with your payment service provider (PSP) to determine the appropriate country code.
```
Comment (vykhrystiuk) [inline openapi/components/schemas/ReadyToPay/Features/PaymentCardDigitalWalletFeatures/ApplePayFeature.yaml:11]:
```suggestion
    description: The country where the payment is processed. This is often the location for settling the payment. Consult with your payment service provider (PSP) to determine the appropriate country code.
```

File: openapi/components/schemas/ReadyToPay/Features/PaymentCardDigitalWalletFeatures/GooglePayFeature.yaml [hunk 4]
```diff
@@ -11,3 +11,9 @@ properties:
     description: The merchant origin in Google Pay. The fully qualified domain name.
     type: string
     example: "www.example.com"
+  countryCode:
+    description: The country where the payment is processed.  This is often the location for settling the payment. Consult with your payment service provider (PSP) to determine the appropriate country code.
```
Comment (vykhrystiuk) [inline openapi/components/schemas/ReadyToPay/Features/PaymentCardDigitalWalletFeatures/GooglePayFeature.yaml:15]:
```suggestion
    description: The country where the payment is processed. This is often the location for settling the payment. Consult with your payment service provider (PSP) to determine the appropriate country code.
```

File: openapi/components/schemas/PaymentCards/DigitalWallets.yaml [hunk 5]
```diff
@@ -16,6 +16,12 @@ properties:
         description: A string of 64 or fewer UTF-8 characters containing the canonical name for your store, suitable for display. Don’t localize the name.
         type: string
         example: "Test Merchant"
+      countryCode:
+        description: The country where the payment is processed. This is often the location for settling the payment. Consult with your payment service provider (PSP) to determine the appropriate country code.
```
Comment (adamaltman) [inline openapi/components/schemas/PaymentCards/DigitalWallets.yaml:20]:
```suggestion
        description: The ISO 3166 alpha-2 country code where the payment is processed. This is often the location for settling the payment. Consult with your payment service provider (PSP) to determine the appropriate country code.
```

File: openapi/components/schemas/ReadyToPay/Features/PaymentCardDigitalWalletFeatures/ApplePayFeature.yaml [hunk 6]
```diff
@@ -7,4 +7,10 @@ properties:
     description: A string of 64 or fewer UTF-8 characters containing the canonical name for your store, suitable for display. Don’t localize the name.
     type: string
     example: "Test Merchant"
+  countryCode:
+    description: The country where the payment is processed. This is often the location for settling the payment. Consult with your payment service provider (PSP) to determine the appropriate country code.
```
Comment (adamaltman) [inline openapi/components/schemas/ReadyToPay/Features/PaymentCardDigitalWalletFeatures/ApplePayFeature.yaml:11]:
```suggestion
    description: The ISO 3166 alpha-2 country code where the payment is processed. This is often the location for settling the payment. Consult with your payment service provider (PSP) to determine the appropriate country code.
```

File: openapi/components/schemas/ReadyToPay/Features/PaymentCardDigitalWalletFeatures/GooglePayFeature.yaml [hunk 7]
```diff
@@ -11,3 +11,9 @@ properties:
     description: The merchant origin in Google Pay. The fully qualified domain name.
     type: string
     example: "www.example.com"
+  countryCode:
+    description: The country where the payment is processed. This is often the location for settling the payment. Consult with your payment service provider (PSP) to determine the appropriate country code.
```
Comment (adamaltman) [inline openapi/components/schemas/ReadyToPay/Features/PaymentCardDigitalWalletFeatures/GooglePayFeature.yaml:15]:
```suggestion
    description: The ISO 3166 alpha-2 country code where the payment is processed. This is often the location for settling the payment. Consult with your payment service provider (PSP) to determine the appropriate country code.
```

**Review by adamaltman — APPROVED** [review]

**Review by akozhevnikov-rebilly — APPROVED** [review]

**Review by vykhrystiuk — APPROVED** [review]

**Review by ashkarpetin — APPROVED** [review]

File: openapi/components/schemas/PaymentCards/DigitalWallets.yaml [hunk 8]
```diff
@@ -16,6 +16,12 @@ properties:
         description: A string of 64 or fewer UTF-8 characters containing the canonical name for your store, suitable for display. Don’t localize the name.
         type: string
         example: "Test Merchant"
+      countryCode:
+        description: The ISO 3166 alpha-2 country code where the payment is processed. This is often the location for settling the payment. Consult with your payment service provider (PSP) to determine the appropriate country code.
+        maxLength: 2
+        type: string
+        readOnly: true
+        example: US
```
Comment (slavcodev) [inline openapi/components/schemas/PaymentCards/DigitalWallets.yaml:24]:
```suggestion
      countryCode:
        $ref: "DigitialWalletCurrencyCode"
```

?

**Pull request merged**